modules/tensor_mechanics/src/materials/ADRadialReturnStressUpdate.C

```diff
@@ -28,7 +28,7 @@ defineADValidParams(
         "effective_inelastic_strain_name",
         "Name of the material property that stores the effective inelastic strain");
     params.addParam<bool>("apply_strain", true, "Flag to apply strain. Used for testing.");
-    params.addParamNamesToGroup("effective_inelastic_strain_name", "Advanced"););
+    params.addParamNamesToGroup("effective_inelastic_strain_name apply_strain", "Advanced"););
```

Comment:
I'm still not a huge fan of this code change in the final code (even if it is helpful for debugging, I'm not fully convinced that it belongs in the final checked in code version); however, I can appreciate and accept this compromise with the advanced parameters group